tools/tidb-binlog-monitor.md

```diff
@@ -3,21 +3,99 @@ title: TiDB-Binlog 监控指标说明
 category: tools
 ---
 
-# TiDB-Binlog 监控指标说明
+# TiDB-Binlog 监控指标及告警说明
+
+本文档介绍 grafana 中 TiDB-Binlog 的各项监控指标说明，以及报警规则说明。
+
+## 监控指标
+
+### Pump
+
+#### Storage Size
+
+- 记录磁盘的总空间大小（capacity），以及可用磁盘空间大小（avaliable）。
+
+#### Metadata
+
+- 记录每个 Pump 的可删除 binlog 的最大 tso（gc_tso），以及保存的 binlog 的最大的 commit tso（max_commit_tso）。
+
+#### Write Binlog QPS by Instance
+
+- 每个 Pump 接收到的写 binlog 请求的 QPS。
```

Comment:
Using either "binlog" or "Binlog" (like in following items) for similar descriptions, but keep them consistent.